pkgs/development/libraries/libdeltachat/default.nix

```
@@ -31,17 +31,17 @@ let
   };
 in stdenv.mkDerivation rec {
   pname = "libdeltachat";
-  version = "1.136.3";
+  version = "1.140.2";
 
   src = fetchFromGitHub {
     owner = "deltachat";
     repo = "deltachat-core-rust";
     rev = "v${version}";
-    hash = "sha256-/ZWpPpxnOCLGswrfbEPvfUn1LpdBQeR5LecRAB0PEhI=";
+    hash = "sha256-BSbvgKiI89B+nxp5McBKTJAwgePt27C1QvSQLhTL7pQ=";
   };
 
   patches = [
-    ./no-static-lib.patch
+    ./0001-no-stabic-lib.patch
```

No need to prefix the patch with `0001-` here.
```suggestion
    ./no-stabic-lib.patch
```

But more importantly, you forgot to delete the outdated patch at `./no-static-lib.patch`.
If you had kept the filename as is, this would likely not have happened.

https://github.com/Mic92/nixpkgs/blob/b839e92bea2d19bea84720ef360ff4a2b9919fc4/pkgs/development/libraries/libdeltachat/no-static-lib.patch

---

I dropped the old name, but can we keep the prefix? This is compatible with `git format-patch`, which I used to generate this patch.

---

I don't think we have policy for how to name patches.
Feel free to keep the prefix.

What made me comment was the fact that the old patch hasn't been deleted and that could have been prevented.

But you fixed that now, thanks. Fine with me.